src/main/kotlin/pl/edu/uj/ii/ksi/mordor/controllers/FilesystemController.kt

```diff
@@ -16,6 +14,8 @@ import pl.edu.uj.ii.ksi.mordor.services.repository.RepositoryDirectory
 import pl.edu.uj.ii.ksi.mordor.services.repository.RepositoryEntity
 import pl.edu.uj.ii.ksi.mordor.services.repository.RepositoryFile
 import pl.edu.uj.ii.ksi.mordor.services.repository.RepositoryService
+import javax.servlet.http.HttpServletRequest
```

Comment:
From what I see, the build failed due to changed import ordering - https://travis-ci.com/KSIUJ/mordor/builds/92484143#L520

Revert the order of imports to make it pass.